docs/data-providers/running-oracle-node.md

```diff
@@ -26,40 +26,70 @@ RedStone consumers will be able to use the data published by your node [on all s
 
 - Make sure to have enough resources to launch a RedStone oracle node, as explained in the [requirements page.](/docs/data-providers/system-requirements)
 - Install [Docker](https://docs.docker.com/get-docker/). You will run the oracle node in a Docker container.
+- Install [Docker Compose](https://docs.docker.com/compose/install/). It will help you manage multiple containers.
+
+### Example Docker Compose configuration
+
+Here's a simple example of a `docker-compose.yml` file that you can use to run the RedStone oracle node locally. You can copy this configuration and adjust it to your needs.
+
+```yaml
+services:
+  redstone-oracle-node:
+    image: public.ecr.aws/y7v2w8b2/redstone-oracle-node:957ca2a1
+    restart: always
+    depends_on:
+      - redstone-kms
+    networks:
+      - public_network
+      - private_network
+    volumes:
+    - redstone-oracle-node:/oracle-node-level-db
+    environment:
+      OVERRIDE_DIRECT_CACHE_SERVICE_URLS: '["https://httpbin.org/anything"]'
+      OVERRIDE_MANIFEST_USING_FILE: ./manifests/dev/dev.json
+      LEVEL_DB_LOCATION: /oracle-node-level-db
+      REMOTE_SIGNER_URL: http://redstone-kms:4499
+      ENABLE_REMOTE_SIGNER: true
+
+  redstone-kms:
+    restart: always
+    image: public.ecr.aws/y7v2w8b2/kms@sha256:6d0adc668d5a9a6d85b2232c6d6f4a5d6966d0faf58bd403ed37efed1e8219f5
+    networks:
+      - private_network
+    expose:
+      - "4499"
+    environment:
+      KMS_PRIVATE_KEY: 0x1111111111111111111111111111111111111111111111111111111111111111
```

Comment:
as I understand we are giving this to data providers as an example ?
putting private key in plaintext in docker-compose is not good example for them imho

Reply:
I'll consider adding this: https://docs.docker.com/compose/how-tos/use-secrets/ but it needs support in node first